README.md

```diff
@@ -29,6 +28,9 @@ It features all common chess mechanisms, including move generation, move validat
 - Architecture Pattern: Model-View-Controller (MVC)
 - Board Representation: Bitboard
 
+This project uses MVC pattern to deploy clean architecture. While some interdependency is unavoidable due to the nature of chess as a game, all such dependencies are contained within each layer and adhere to SOLID principles.
+![MVC](https://i.imgur.com/wci7Tt4.png)
```

Comment:
This image should be changed. See comment.

Reply:
Changed

Reply:
Looks great. The diagram demonstrates our MVC-Clean Architecture implementation.